test/Hintman/Suggest.hs

```diff
@@ -0,0 +1,57 @@
+{-# LANGUAGE NoImplicitPrelude #-}
```

Comment:
You don't need this extesion. We're using `base-noprelude` so you also don't need to reexport `Relude` module. Should just work